packages/mini-apps-ui-kit-react/src/components/Marble/Marble.tsx

```diff
@@ -0,0 +1,27 @@
+import { cn } from "@/lib/utils";
+import { ComponentProps, forwardRef } from "react";
+
+interface MarbleProps extends ComponentProps<"div"> {
+  imageUrl: string;
+}
+
+export const Marble = forwardRef<HTMLDivElement, MarbleProps>(
+  ({ imageUrl, className, ...props }, ref) => {
+    return (
+      <div
+        ref={ref}
+        className={cn(
+          "rounded-full overflow-hidden border-2 border-gray-100 p-[3px] aspect-square w-[120px]",
+          className,
+        )}
+        {...props}
+      >
+        <img src={imageUrl} alt="Marble" className="object-cover size-full rounded-full" />
+      </div>
+    );
+  },
+);
+
+Marble.displayName = "Marble";
+
+export default Marble;
```

Comment:
```suggestion
import { cn } from "@/lib/utils";
import { ComponentProps, forwardRef } from "react";

export const Marble = forwardRef<HTMLImageElement, ComponentProps<"img">>((props, ref) => {
  return (
    <img
      ref={ref}
      alt="Marble"
      className={cn(
        "rounded-full border-2 border-gray-100 p-[3px] aspect-square w-[120px] object-cover",
        props.className,
      )}
      {...props}
    />
  );
});

Marble.displayName = "Marble";

export default Marble;
```

wdyt of the marble being a just an `<img />` tag? For simplicity and to accept image attributes.

Reply:
Sounds good!